### ISSUE TITLE
How to change maximum of Y-axis in chart.png?

### ISSUE THREAD

**Issue by xic10** [AlexeyAB/darknet#5542]
I am training on the bdd100k dataset with this command on colab.
! ./darknet detector train data/obj.data yolo-obj.cfg backup/yolo-obj_last.weights -dont_show -mjpeg_port 8090 -map
My loss fluctuated in range (10, 15). The reason seems to be that one class only appears less than 1% of the train set. Is there a way to change the maximum of the Y-axis or to reduce the loss?

Also, during training, the print looks like
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.813452, GIOU: 0.807268), Class: 0.763082, Obj: 0.799886, No Obj: 0.006740, .5R: 1.000000, .75R: 0.777778, count: 9, class_loss = 1.523111, iou_loss = 2.446079, total_loss = 3.969190 
Does the v3 mean that I am training on yolov3 instead of v4?

**Comment by AlexeyAB** [comment]
>  Is there a way to change the maximum of the Y-axis or to reduce the loss?

I will add a `max_loss_chart=` param later.

> Does the v3 mean that I am training on yolov3 instead of v4?

No.

**Comment by tianyuluan** [comment]
> > Is there a way to change the maximum of the Y-axis or to reduce the loss?
> 
> I will add a `max_loss_chart=` param later.
> 
> > Does the v3 mean that I am training on yolov3 instead of v4?
> 
> No.

hi!can a way to change the maximum of the y-axis be released? thanks very much!

**Comment by AlexeyAB** [comment]
Use this param with the latest Darknet version:
```
[net]
max_chart_loss = 10
```
